docker/.env

```diff
@@ -1,4 +1,4 @@
 w_postgres_user=testUser
 w_postgres_password=testPassword
 w_connection_string="Host=postgres_db_backend;Database=GlobalCodersApi;Username=testUser;Password=testPassword"
-w_file_storage_base=/tmp/storage/
\ No newline at end of file
+w_file_storage_base="C:/Users/alper/OneDrive/Masaüstü/sd"
```

Comment:
Can you roll back this changes? 
Just use this setting on your machine, but do not push them to git :)